Mathlib/Algebra/FreeMonoid/Basic.lean

```diff
@@ -254,4 +377,68 @@ instance uniqueUnits : Unique (FreeMonoid α)ˣ where
     have : toList u.val ++ toList u.inv = [] := DFunLike.congr_arg toList u.val_inv
     (List.append_eq_nil.mp this).1
 
+@[to_additive]
+theorem map_surjective {f : α → β} : Function.Surjective f → Function.Surjective (map f) := by
```

Comment:
```suggestion
@[to_additive (attr := simp)]
theorem map_surjective {f : α → β} : Function.Surjective (map f) ↔ Function.Surjective f := by
```

Reply:
done! (i can't hit "commit suggestion" since I needed to prove the other implication, but it's in there now)